### PR TITLE
#261 assignments are not sorted sometime

### PR DESCRIPTION
**Pull request by pushyamig**
#261

### CODE REVIEW

File: dashboard/views.py [hunk 1]
```diff
@@ -312,7 +312,7 @@ def assignment_view(request, course_id=0):
     week_list = []
     for item in assignment_list:
         week_list.append(item['week'])
-    weeks = set(week_list)
+    weeks = sorted(set(week_list))
```
Comment (jonespm) [inline dashboard/views.py]: Why is there a new set variable here? Seems like it should have just been
week_list.sort()

And use week_list later in the for loop.

```suggestion
    week_list.sort()
```

And changing
`for week in week_list:`

Comment (jonespm) [reply dashboard/views.py]: Is this just a set to remove duplicates?

Comment (pushyamig) [reply dashboard/views.py]: Yes to remove the duplicated it is a `Set`

Comment (jonespm) [reply dashboard/views.py]: Yeah, so then using set from the beginning probably would have made the code a little easier to read. https://github.com/tl-its-umich-edu/student-dashboard-django/compare/master...jonespm:issue_261-rewrite?expand=1

Yeah, minor cleanup but this file could probably a little more cleanups.

Comment (jonespm) [reply dashboard/views.py]: Also another cool python function is [enumerate](http://book.pythontips.com/en/latest/enumerate.html) which I changed the for loop below to use
https://github.com/tl-its-umich-edu/student-dashboard-django/commit/54afbb42ce6edb451ecf5a3aa5c38bf9bfe922c5

**Pull request merged**